src/collection.js

```diff
@@ -989,7 +989,7 @@ export default class Collection {
     const serverChanged = unquoted > options.lastModified;
     const emptyCollection = data.length === 0;
     if (!options.exclude && localSynced && serverChanged && emptyCollection) {
-      throw Error("Server has been flushed.");
+      throw Error("Server has been flushed." + since);
```

Comment:
Thank you for your proposal. That's a good start.

However this is a bit too simple, no space between the dot and the value, we don't actually know what the since value is while reading the message. Is it the server timestamp or the client timestamp?

> I guess the lastModified from the client as well as the one the server responded with would be good.

Can you enhance your PR by adding commits to it and display both the server side timestamp and the client side one?

I would also recommend adding a test for it.

Reply:
The test you are looking to contribute to is here: https://github.com/Kinto/kinto.js/blob/571b0aeb2cc69f67d86523133d46b332cca5b3ae/test/integration_test.js#L2001

Reply:
Hi, so I fixed the issue with the client and the server timestamp, how would I go about matching the error to the error message in the integration_test file?

Reply:
You can use a regular expression instead of a literal string to match the error message.